pkg/crc/services/dns/template.go

```diff
@@ -35,6 +37,21 @@ const (
 	zoneFileAddressTemplate = `; Name Server Information
 _etcd-server-ssl._tcp.{{ .ClusterName}}.{{ .BaseDomain }}. IN SRV 10 10 2380 etcd-0.{{ .ClusterName }}.{{ .BaseDomain }}.
 *.{{ .BaseDomain }}.                IN  A     {{ .IP }}
+`
+
+	dnsmasqConfTemplate = `user=root
```

Comment:
Let me verify this tomorrow and also compare with what we did before https://github.com/minishift/minishift/blob/master/pkg/minishift/network/dns/localdns_docker.go#L29-L51